ddtrace/tracer/metrics.go

```diff
@@ -85,7 +85,9 @@ func (t *tracer) reportRuntimeMetrics(interval time.Duration) {
 	}
 }
 
-func (t *tracer) reportHealthMetrics(interval time.Duration) {
+// reportHealthMetricsAtInterval reports health metrics at the specified interval to reduce noise.
```

Comment:
Smallest nit in the world: since we are clarifying that not all health metrics are contained in this function, should we also be clear about that in this comment? Something along the lines of:

```suggestion
// reportHealthMetricsAtInterval reports certain/noisy health metrics at the specified interval to reduce noise.
```

I'm not sure what word will best describe these metrics, but dealer's choice 👍🏼